keps/sig-apps/2214-indexed-job/README.md

```diff
@@ -424,64 +424,60 @@ _This section must be completed when targeting beta graduation to a release._
 
 * **What specific metrics should inform a rollback?**
 
+  - job_sync_duration_seconds shows significantly more latency for Indexed Jobs.
+  - job_sync_total shows errors for Indexed Jobs only.
+  - job_finished_total shows that Indexed Jobs don't finish.
```

Comment:
Any particular reasons why these metrics will be for Indexed Jobs only? IMO it seems reasonable to expose them for all jobs.

Reply:
I don't plan to add them just for Indexed Jobs, but there will be a label for the mode.

Reply:
Updated to reflect that users can use a label to compare Indexed to NonIndexed